src/router_core/exchange_bindings.c

```diff
@@ -16,14 +16,16 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-#include <qpid/dispatch/ctools.h>
-#include <inttypes.h>
-#include <stdio.h>
-#include "router_core_private.h"
-#include "forwarder.h"
 #include "exchange_bindings.h"
```

Comment:
Here's the only issue the reordering found. `exchange_bindings.h` depended on being included after `"router_core_private.h"`.